### ISSUE TITLE
[Bug] Steam feeds have duplicate stories

### ISSUE THREAD

**Issue by Splike** [samuelclay/NewsBlur#1278]
All my Steam RSS feeds for different games have duplicate stories. One story with the actual content, and another with the same story title, but the contents just say:
```
: You can find the details for this event on the announcement page [url=https://steamcommunity.com/ogg/1145360/announcements/detail/1705110618940067328]here[/url].
```
Example feed:
RSS URL: https://steamcommunity.com/games/1145360/rss
Website URL: https://steamcommunity.com/games/1145360

**Comment by samuelclay** [comment]
Not sure what I can do about it as the item ids are different, as are the urls and story content.

**Comment by Splike** [comment]
The titles are exactly the same and and the stories with proper content always have an author attached, whereas the bogus ones do not.

Maybe it's possible to filter out all stories that do not have an author, has a title that matches another in the feed, and has content that matches this string (URL numbers of course vary).
`: You can find the details for this event on the announcement page [url=https://steamcommunity.com/ogg/1145360/announcements/detail/1705110618940067328]here[/url].`

and then apply this filtering to stories on all steam RSS feeds of the form `https://steamcommunity.com/games/[GAME ID]/rss`

**Comment by samuelclay** [comment]
Training on stories with no author is a fascinating idea. How would that UI look? Just an option on the full size story trainer, pictured below... 

<img width="876" alt="Screen Shot 2020-02-01 at Feb 1 11 16 48 AM" src="https://user-images.githubusercontent.com/44229/73595244-5a832d80-44e4-11ea-9ffc-25d058130366.png">


Reopening an turning into that idea.

**Comment by samuelclay** [comment]
Add it to the "Everything by this publisher" group as "Missing author" and save it as a `authors` classifier

**Comment by Splike** [comment]
Currently, stories that have an author has a section in training called "Story Author" and stories without an author doesn't have this section.

Instead of adding "Missing author" under "Everything by this Publisher", it would make sense to add it under "Story Author", so all stories, author or not, will have this section.

**Comment by Splike** [comment]
I'd like to bump this issue, as it's still something I'm dealing with 2 years later. My suggestion above still stands as the solution I think is most appropriate.

**Comment by samuelclay** [comment]
I agree that this is still a good idea to implement, but it's still not high enough of a priority. I would need to add the UI as well as handle a missing author everywhere story counts are made, which is why I'm hesitant to implement this.

I'll say this, I have ideas for a more customizable filter that allows you to apply regex to stories, and that would work for this use case. But it would only be available to the Pro tier, which I'm pricing at $20/month.

**Comment by samuelclay** [comment]
Consolidating into #1880 — both issues request the ability to use the intelligence classifier/trainer on stories with empty or absent author fields. The underlying fix is the same: the classifier currently short-circuits on empty authors in `apply_classifier_authors()`.
